code/src/containers/tables/orders-table/components/orders-table-body/OrdersTableBody.tsx

```diff
@@ -28,26 +31,61 @@ const OrdersTableBody = ({ order }: OrderTableBodyProps) => {
     isPaid
   } = order;
 
-  const orderItemStatus = orderStatusesTranslationKeys[orderStatus];
   const orderReceiver = `${firstName} ${lastName}`;
 
-  const orderBadgeItemStatus = (
-    <AppTypography
-      className="spa-order-table__body-status"
-      variant="caption"
-      translationKey={orderItemStatus}
-    />
+  const statusBlock = (
+    <AppSelect
+      defaultValue={orderStatus}
+      value={orderStatus}
+      IconComponent={() => null}
```

Comment:
Is `() => null` required here?

Reply:
I it is needed to set icon to null. Just `null` doesn't work with ts. Not sure if there is other way